jdi-light-angular-tests/src/main/java/io/github/com/pages/sections/SliderSection.java

```diff
@@ -0,0 +1,51 @@
+package io.github.com.pages.sections;
+
+import com.epam.jdi.angular.elements.common.Checkbox;
+import com.epam.jdi.light.elements.common.UIElement;
+import com.epam.jdi.light.elements.composite.Section;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.angular.elements.common.Slider;
+import com.epam.jdi.light.ui.html.elements.common.TextField;
+
+
+public class SliderSection extends Section {
```

Comment:
SliderSection should consist of all AngularPage sliders: basic, configurable, formatting slider and the related buttons, labels, etc